src/Libraries/Nop.Services/Installation/CodeFirstInstallationService.cs

```diff
@@ -6254,6 +6254,7 @@ protected virtual void InstallSettings()
                 ShowShareButton = true,
                 PageShareCode = "<!-- AddThis Button BEGIN --><div class=\"addthis_toolbox addthis_default_style \"><a class=\"addthis_button_preferred_1\"></a><a class=\"addthis_button_preferred_2\"></a><a class=\"addthis_button_preferred_3\"></a><a class=\"addthis_button_preferred_4\"></a><a class=\"addthis_button_compact\"></a><a class=\"addthis_counter addthis_bubble_style\"></a></div><script src=\"http://s7.addthis.com/js/250/addthis_widget.js#pubid=nopsolutions\"></script><!-- AddThis Button END -->",
                 ProductReviewsMustBeApproved = false,
+                OneReviewPerProductFromCustomer = false,

```

Comment:
actually this is not needed as a boolean by default is having false value, but for keeping consistency added default